jormungandr/Cargo.toml

```diff
@@ -36,10 +36,11 @@ hex = "0.4"
 http = "0.2.2"
 humantime = "2.0"
 jormungandr-lib = { path = "../jormungandr-lib" }
+keynesis = "1.1"
```

Comment:
Is a new dependency for `ed25519` keys strictly required?

Reply:
`poldercast` use key types from `keynesis`, so we need to import that library if we want to interact with it (e.g. conversion between 'our' type and their and implementing custom layers).